themes/website/templates/clients.html

```diff
@@ -82,9 +82,6 @@ <h2 class="header">Ricin</h2>
 
 				<table class="table table-condensed">
 					<tbody>
-						<tr>
-							<th>Website:</th>
-							<th><a href="https://ricin.im">https://ricin.im</a>
 						<tr>
```

Comment:
Instead of removing the URL, maybe change it to https://konv.im as it stills provides valid downloads?